README.md

```diff
@@ -10,3 +10,19 @@ scrape turbovax site and alert when there is availability in NYC area
   python setup.py build 
   python setup.py install 
 ```
+# Install GeckoDriver on Linux
+```bash
+wget https://github.com/mozilla/geckodriver/releases/download/v0.29.0/geckodriver-v0.29.0-linux64.tar.gz
+wget https://github.com/mozilla/geckodriver/releases/download/v0.29.0/geckodriver-v0.29.0-macos.tar.gz
```

Comment:
lets try -e 'http.github....'
for example:

[-e] git+https://git.example.com/MyProject.git@master#egg=MyProject

https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format

would this work?

Reply:
maybe this would do? https://pypi.org/project/geckodriver-autoinstaller/